lib/dataContract/DataContract.js

```diff
@@ -204,6 +204,22 @@ class DataContract {
     return this.binaryProperties[type];
   }
 
+  /**
+   * Set metadata
+   * @param {Object} metadata
+   */
+  setMetadata(metadata) {
```

Comment:
We need to define a class for metadata. Atm, we need `blockHeight` and `coreChainLockedHeigth`.